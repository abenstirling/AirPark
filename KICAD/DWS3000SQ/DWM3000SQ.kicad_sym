
(kicad_symbol_lib (version 20211014) (generator kicad_symbol_editor)
  (symbol "DWM3000SQ" (pin_names (offset 1.016)) (in_bom yes) (on_board yes)
    (property "Reference" "U" (id 0) (at -22.86 18.542 0)
      (effects (font (size 1.27 1.27)) (justify bottom left))
    )
    (property "Value" "DWM3000SQ" (id 1) (at -22.86 -22.86 0)
      (effects (font (size 1.27 1.27)) (justify bottom left))
    )
    (property "Footprint" "XCVR_DWM3000SQ" (id 2) (at 0 0 0)
      (effects (font (size 1.27 1.27)) (justify bottom) hide)
    )
    (property "MAXIMUM_PACKAGE_HEIGHT" "2.89mm" (id 4) (at 0 0 0)
      (effects (font (size 1.27 1.27)) (justify bottom) hide)
    )
    (property "PARTREV" "B" (id 5) (at 0 0 0)
      (effects (font (size 1.27 1.27)) (justify bottom) hide)
    )
    (property "MANUFACTURER" "QORVO" (id 6) (at 0 0 0)
      (effects (font (size 1.27 1.27)) (justify bottom) hide)
    )
    (property "SNAPEDA_PN" "DWM3000TR13" (id 7) (at 0 0 0)
      (effects (font (size 1.27 1.27)) (justify bottom) hide)
    )
    (property "STANDARD" "Manufacturer Recommendations" (id 8) (at 0 0 0)
      (effects (font (size 1.27 1.27)) (justify bottom) hide)
    )
    (symbol "DWM3000SQ_0_0"
      (rectangle (start -22.86 -20.32) (end 22.86 17.78)
        (stroke (width 0.254)) (fill (type background))
      )
      (pin output line (at 27.94 5.08 180.0) (length 5.08)
        (name "EXTON"
          (effects (font (size 1.016 1.016)))
        )
        (number "1"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin bidirectional line (at 27.94 -12.7 180.0) (length 5.08)
        (name "WAKEUP"
          (effects (font (size 1.016 1.016)))
        )
        (number "2"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin output line (at 27.94 7.62 180.0) (length 5.08)
        (name "~{RST}"
          (effects (font (size 1.016 1.016)))
        )
        (number "3"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin power_in line (at 27.94 15.24 180.0) (length 5.08)
        (name "VDD1"
          (effects (font (size 1.016 1.016)))
        )
        (number "5"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin power_in line (at 27.94 12.7 180.0) (length 5.08)
        (name "VDD3V3"
          (effects (font (size 1.016 1.016)))
        )
        (number "6"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin power_in line (at 27.94 12.7 180.0) (length 5.08)
        (name "VDD3V3"
          (effects (font (size 1.016 1.016)))
        )
        (number "7"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin power_in line (at 27.94 -17.78 180.0) (length 5.08)
        (name "VSS"
          (effects (font (size 1.016 1.016)))
        )
        (number "8"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin power_in line (at 27.94 -17.78 180.0) (length 5.08)
        (name "VSS"
          (effects (font (size 1.016 1.016)))
        )
        (number "16"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin power_in line (at 27.94 -17.78 180.0) (length 5.08)
        (name "VSS"
          (effects (font (size 1.016 1.016)))
        )
        (number "21"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin power_in line (at 27.94 -17.78 180.0) (length 5.08)
        (name "VSS"
          (effects (font (size 1.016 1.016)))
        )
        (number "23"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin power_in line (at 27.94 -17.78 180.0) (length 5.08)
        (name "VSS"
          (effects (font (size 1.016 1.016)))
        )
        (number "24"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin bidirectional line (at -27.94 -12.7 0) (length 5.08)
        (name "GPIO8/IRQ"
          (effects (font (size 1.016 1.016)))
        )
        (number "22"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin input clock (at 27.94 -2.54 180.0) (length 5.08)
        (name "SPICLK"
          (effects (font (size 1.016 1.016)))
        )
        (number "20"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin output line (at 27.94 -5.08 180.0) (length 5.08)
        (name "SPIMISO"
          (effects (font (size 1.016 1.016)))
        )
        (number "19"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin input line (at 27.94 -7.62 180.0) (length 5.08)
        (name "SPIMOSI"
          (effects (font (size 1.016 1.016)))
        )
        (number "18"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin input line (at 27.94 0.0 180.0) (length 5.08)
        (name "~{SPICS}"
          (effects (font (size 1.016 1.016)))
        )
        (number "17"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin bidirectional line (at -27.94 7.62 0) (length 5.08)
        (name "GPIO0/RXOKLED"
          (effects (font (size 1.016 1.016)))
        )
        (number "15"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin bidirectional line (at -27.94 5.08 0) (length 5.08)
        (name "GPIO1/SFDLED"
          (effects (font (size 1.016 1.016)))
        )
        (number "14"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin bidirectional line (at -27.94 2.54 0) (length 5.08)
        (name "GPIO2/RXLED"
          (effects (font (size 1.016 1.016)))
        )
        (number "13"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin bidirectional line (at -27.94 0.0 0) (length 5.08)
        (name "GPIO3/TXLED"
          (effects (font (size 1.016 1.016)))
        )
        (number "12"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin bidirectional line (at -27.94 -2.54 0) (length 5.08)
        (name "GPIO4"
          (effects (font (size 1.016 1.016)))
        )
        (number "11"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin bidirectional line (at -27.94 -5.08 0) (length 5.08)
        (name "GPIO5/EXTTXE/SPIPOL"
          (effects (font (size 1.016 1.016)))
        )
        (number "10"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin bidirectional line (at -27.94 -7.62 0) (length 5.08)
        (name "GPIO6/EXTRXE/SPIPHA"
          (effects (font (size 1.016 1.016)))
        )
        (number "9"
          (effects (font (size 1.016 1.016)))
        )
      )
      (pin bidirectional line (at -27.94 -10.16 0) (length 5.08)
        (name "GPIO7/SYNC"
          (effects (font (size 1.016 1.016)))
        )
        (number "4"
          (effects (font (size 1.016 1.016)))
        )
      )
    )
  )
)