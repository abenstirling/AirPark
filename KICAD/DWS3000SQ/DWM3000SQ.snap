{"u_token":"SAXJK1cdp5bonGKzWxkg2m7vRwd2pIYz","part_number":"DWM3000SQ","manufacturer":"Qorvo","mode":"place-part","plugin":".snap","source":"https://www.snapeda.com"}